File: PalindromeNumber/palindromenumber.py
class Solution:
    def isPalindrome(self, x: int) -> bool:
        
        if x > 0 or x == 0:
            # if integer is positive reverse int
            reversed = int(str(x)[::-1])
            if reversed == x:
                return True 
            else:
                return False
        else:
            return False
        
'''
Runtime beats 89.67 % of python3 submissions.
memory usage beats 47.89 % of python3 submissions.
Determine whether an integer is a palindrome. An integer is a palindrome when it reads the same backward as forward.
'''
        
       

File: PalindromeNumber/palindromenumber.php
class Solution {

    /**
     * @param Integer $x
     * @return Boolean
     */
    function isPalindrome($x) {
        $xRev = intval(strrev((string)$x));
        if($xRev != $x){
            return false;
        }
        elseif($xRev == $x){
            return true;
        }
    }
}

// Runtime: 24 ms, faster than 90.91% of PHP online submissions for Palindrome Number.
// Memory Usage: 14.8 MB, less than 43.33% of PHP online submissions for Palindrome Number.
